tst/providers/MeetingProvider/index.test.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {LogLevel, EventName, EventAttributes, DefaultEventController, NoOpDebugLogger, MeetingSessionConfiguration, MeetingSessionCredentials, MeetingSessionURLs} from "amazon-chime-sdk-js";
```

Comment:
I think we do not lint the `tst` folder but can you once run it and see if it formats correctly? These imports are in-consistently formatted.

Reply:
Ran it but didn't catch this so I manually updated it.